RyanYoung25/CS377
# Script to populate database with sample data

### Changes
- makefile to setup environment and run script to populate db with sample data
- Adds two test users test@email.com, test2@email.com to Users table
- Adds a calendar and timeslots for test@email.com in Calendars and TimeSlots tables respectively

### How to Test
1. Run with make. Enter your password (if you have one)
2. Check with mysql server client that test data is in DB
3. Open app and login to see calendars
